tests/engine/test_multiproc_workers.py

```diff
@@ -99,14 +99,13 @@ def execute_workers(worker_input: str) -> None:
         assert isinstance(e, ChildProcessError)
 
 
+# UPSTREAM SYNC
```

Comment:
Do we know why this is failing on 3.8?

Reply:
No, I have not figured that out yet. It passes locally in py38 when running both the individual test as well as the entire test file.

Reply:
okay - let's pull this out then

I think that might have been a spurious failure. Can add back in later if needed

Reply:
Done. It has failed at least a couple of times, but I think it’s included in remote-push so we can see on this PR itself if it passes or not.